internal/events/event.go

```diff
@@ -272,7 +272,7 @@ func filterEvents(tx *gorm.DB, clusterID *strfmt.UUID, hostIds []strfmt.UUID, in
 
 		// filter by event message
 		if message != nil {
-			tx = tx.Where("events.message LIKE ?", fmt.Sprintf("%%%s%%", *message))
+			tx = tx.Where("LOWER(events.message) LIKE ?", processEventMessage(*message))
```

Comment:
```suggestion
			tx = tx.Where("LOWER(events.message) LIKE %?%", quoteQueryPlaceHolders(*message))
```
I think this would make the intent clearer, WDYT?

Reply:
I think GORM doesn't take `%?%` well